donkeycar/tests/_test_train.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*- #
```

Comment:
Well, this change looks weird :)

Reply:
This is just an archived change.

Reply:
Just delete not needed files, they are in git history anyway.